### ISSUE TITLE
Trouble re-loading Doc2Vec Model Generated In earlier Version

### ISSUE THREAD

**Issue by fsfeng** [RaRe-Technologies/gensim#1977]
I trained a Doc2Vec model in an earlier version of gensim and now can not load it. I tried both `Doc2Vec.load(fname)` and `Doc2Vec.load(fname, mmap = 'r')`. Both produce the same errors:

`AttributeError: Can't get attribute 'DocvecsArray' on <module 'gensim.models.doc2vec' from 'DIR/python/miniconda3/lib/python3.6/site-packages/gensim/models/doc2vec.py'>`

` AttributeError: 'Doc2Vec' object has no attribute 'running_training_loss'`

Any help would be much appreciated. Thanks!

**Comment by menshikh-iv** [comment]
Hello, can you share your model (or install current `develop` branch and check again), probably this already resolved in #1974  (but I'm not sure)

**Comment by fsfeng** [comment]
Hi I tried with the `develop` branch install but am getting the exact same problem.

Not sure what you mean by sharing my model, but here's the code I used to create it.

```
from gensim import models
from gensim.models.doc2vec import TaggedDocument

class DocIterator(object):
    def __init__(self, doc_list, labels_list):
        self.labels_list = labels_list
        self.doc_list = doc_list

    def __iter__(self):
        for idx, doc in enumerate(self.doc_list):
            # print 'creating tagged document...%d' % idx
            yield TaggedDocument(words=doc.split(), tags=[self.labels_list[idx]])


iterator = DocIterator(TEXT, LABELS)

%time model = gensim.models.Doc2Vec(size=100, window=10, min_count=50, workers=4, alpha=0.025, min_alpha=0.025)
%time model.build_vocab(iterator)

print('done building vocabulary')
print('start training the model')

for epoch in range(10):
    starttime = datetime.datetime.now()
    print("start", starttime)
    print('epoch %d..' % (epoch+1))
    model.train(iterator)
    model.alpha -= 0.002
    model.min_alpha = model.alpha
    model.train(iterator)
    endtime = datetime.datetime.now()
    print("end", endtime, "took", endtime-starttime)

model.save("DIR/doc2vec_1030.model")
```
This was done a few months ago and now when I try to re-load the model I'm getting the error reported above.

**Comment by menshikh-iv** [comment]
@fsfeng about "sharing model" I mean load it `doc2vec_1030.model` to GitHub/google drive/etc and publish a link here (this needed for investigation & fixing).

What's a gensim version you are used?

CC: @manneshiva probably this is a bug, please fix it ASAP.

**Comment by marlohaering** [comment]
I encountered the same error. I trained my model with gensim version 1.0.1 and I'm trying to reload it with gensim version 3.4.0. Unfortunately, I cannot share my model.

**Comment by menshikh-iv** [comment]
@marlohaering that's expected behavior, we don't have so long backward compatibility for models.

**Comment by menshikh-iv** [comment]
@fsfeng I'm still waiting for a model from you (this needed for reproducing an error), also, what's gensim version you used for training?

**Comment by fsfeng** [comment]
Hi, I fixed this just by re-installing Gensim version 2.3.

**Comment by fsfeng** [comment]
I also cannot share the model unfortunately, all is stored on a secure
university virtual server.

**Comment by menshikh-iv** [comment]
@fsfeng thanks for the information, now I understand what happens.

**Comment by menshikh-iv** [comment]
fixed in #2012
